.azure/scripts/setup-kubernetes.sh

```diff
@@ -78,47 +142,108 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
     fi
 
     minikube addons enable default-storageclass
+    add_docker_hub_credentials_to_kubernetes "$TEST_CLUSTER"
+    setup_registry_proxy
+
+elif [ "$TEST_CLUSTER" = "kind" ]; then
```

Comment:
Would it make sense to have two separate scripts for `kind` and `minikube` setup, to make this script more clear? I guess that it can be confusing having both in same script file.

Reply:
Well, there is a lot of shared code between those two installations. So I wanted to have it as one script. I could make a `common.sh` and divide these 2 scripts, but I don't know how much value it gives. But if you really want it I can divide it :)

Reply:
I have divided them into two separate scripts ;)